src/components/DailyStatistics/TitleStatistic.tsx

```diff
@@ -71,7 +76,9 @@ export const TitleStatisticComponent: FunctionComponent<TitleStatisticComponent>
           color: theme.statisticsScreen.smallTextColor,
         }}
       >
-        {i18nt("statisticsScreen", "distributedAmount")}
+        {theme.name === GOVWALLET_THEME_NAME
```

Comment:
@dalsontws is this the approach you envisioned for the use of the theme's name? How does it integrate with the campaign specific translations, i.e. c13n?

Reply:
not really, this was used initially to ensure minimal changes to the existing campaigns' config. do you this can be achieved without doing so?

Reply:
@yao-peijun you can refer to this PR [here](https://github.com/rationally-app/mobile-application/pull/508) on how we use `c13nt` for campaign specific translations!